src/Humanizer/Localisation/Formatters/DefaultFormatter.cs

```diff
@@ -88,22 +88,34 @@ private string GetResourceForTimeSpan(TimeUnit unit, int count)
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="resourceKey"></param>
+        /// <param name="resourceKey">The resource key.</param>
         /// <returns></returns>
+        /// <exception cref="ArgumentException">If the resource not exists on the specified culture.</exception>
         protected virtual string Format(string resourceKey)
         {
-            return Resources.GetResource(GetResourceKey(resourceKey), _culture);
+            var resourceString = Resources.GetResource(GetResourceKey(resourceKey), _culture);
+
+            if (string.IsNullOrEmpty(resourceString))
+                throw new ArgumentException(string.Format("The resource object with key '{0}' was not found", resourceKey), nameof(resourceKey));
+
+            return resourceString;
         }
 
         /// <summary>
-        /// 
+        /// Formats the specified resource key.
         /// </summary>
-        /// <param name="resourceKey"></param>
-        /// <param name="number"></param>
+        /// <param name="resourceKey">The resource key.</param>
+        /// <param name="number">The number.</param>
         /// <returns></returns>
+        /// <exception cref="ArgumentException">If the resource not exists on the specified culture.</exception>
         protected virtual string Format(string resourceKey, int number)
         {
-            return Resources.GetResource(GetResourceKey(resourceKey, number), _culture).FormatWith(number);
+            var resourceString = Resources.GetResource(GetResourceKey(resourceKey, number), _culture);
+
+            if (string.IsNullOrEmpty(resourceString))
+                throw new ArgumentException(string.Format("The resource object with key '{0}' was not found", resourceKey), nameof(resourceKey));
```

Comment:
> Unhandled Exception: System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt

I don't think so that was me :D Should I re-trigger?